CONTRIBUTING.md

```
@@ -83,6 +83,31 @@ Jest snapshot testing might seems like a quick and easy solution to test your Re
 
 This repository is using the [Conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) message format. Please follow the specification.
 
+### Scope requirements
+
+Because this repo uses `useCommitScope: true` for Nx versioning, **versioning commits** (`feat`, `fix`, or commits with a `!` breaking-change marker) must use the **full Nx project name** as the scope:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)